scripts/reindex.py

```diff
@@ -50,16 +50,17 @@
                     default=defaults.num_workers,
                     type=int,
                     help='The number of workers that will be sending bundles to the indexer concurrently')
-parser.add_argument('--partition-prefix-length',
-                    metavar='NUM',
-                    default=0,
-                    type=int,
-                    help='The length of the bundle UUID prefix by which to partition the set of bundles matching the '
-                         'query. Each query partition is processed independently and remotely by the indexer lambda. '
-                         'The lambda queries the repository and queues a notification for each matching bundle. If 0 '
-                         '(the default) no partitioning occurs, the repository is queried locally and the indexer '
-                         'notification endpoint is invoked for each bundle individually and concurrently using worker'
-                         'threads. This is magnitudes slower than remote i.e., partitioned indexing.')
+parser.add_argument('--remote',
+                    dest='remote',
+                    default=False,
+                    action='store_true',
```

Comment:
Since `--remote` is the normal case, I think we should

 ```suggestion
parser.add_argument('--local',
                    dest='local',
                    default=False,
                    action='store_true',
```